src/views/forms/AccountManagementForms/WithdrawForm.tsx

```diff
@@ -111,7 +111,12 @@ export const WithdrawForm = () => {
     const prefix = exchange ? 'noble' : grazChainPrefix;
     return isValidAddress({
       address: toAddress,
-      network: chainIdStr === 'solana' ? 'solana' : prefix ? 'cosmos' : 'evm',
+      network:
```

Comment:
can we follow the pattern we do here https://github.com/dydxprotocol/v4-web/blob/main/src/constants/graz.ts#L33-L35 so we can use the right address in other places in the codebase :D

Reply:
e.g.:
```
export const getSolanaChainId = () => isMainnet ? 'solana' : 'solana-devnet'
```

```
network: chainIdStr === getSolanaChainId() ? 'solana'
...etc etc
```

Reply:
chatted offline, just gonna roll this into the rewrite